docs/js/faq.md

```diff
@@ -0,0 +1,96 @@
+
+### Can I use features annotated as Beta in production?
+
+We __do not recommend__ using API that is marked unstable in productive code. We do not guarantee any API compatibility for future updates and the features might be experimental. You can use these features to test cutting edge functionality, provide us feedback, and plan migration steps when _Beta_ features are releases as _General
+Availability_.
+
+### I think I found a bug in the SDK, what should I do?
+
+Please, report it to us via any available channel. The preferred support channel is [Github](https://github.com/SAP/cloud-sdk/issues/new/choose). You can also ask a question on [Stack
+Overflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk ).
+
+### I'm creating a BCP incident, what's your component name?
+
+Choose `XX-S4C-SDK` if you are reporting an issue via BCP.
+
+## OData related questions ##
+
+### What versions of OData protocol do you support?
+
+We support OData v2 and OData v4 services. You can use pre-generated client libraries supplied with SDK or generate your client from the SDK specification. [Find more details here.](features/odata/overview)
+
+### Do you support ALL OData features?
+
+We support most of the OData features that are exposed by SAP services. However, the [OData specification](https://www.odata.org/documentation/) is huge and contains many features that would see rare to no use. If you found a feature that you need but it is not yet supported by Cloud SDK for Java, please, make a feature request via [Github](https://github.com/SAP/cloud-sdk/issues/new/choose)
+
+### I receive an OData error/exception when using Cloud SDK for JavaScript
+
+You may see some errors while developing. These errors are not always caused by flaws in the SDK as we often see inconsistent OData protocol handling by different services. Some of them even have known flaws for which we have workarounds. If you can't solve your issue via debugging and experimenting, please, report your incident via [Stack Overflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk ) or [Github](https://github.com/SAP/cloud-sdk/issues/new/choose).
+
+## REST related questions ##
+
+### Do you support REST client libraries?
+
+We're working on it. The current support in experimental and not ready for production use.
+
+## Questions about SAP Cloud Platform
+
+### Do you support SAP Cloud Platform - Cloud Foundry?
+
+SAP Cloud SDK for JavaScript has first-class support for [SCP Cloud
+Foundry](https://www.sap.com/products/cloud-platform.html). We provide plenty of helpful abstractions for [connectivity](features/connectivity/destination-js-sdk) and authentication that make developing Apps a pleasant and rewarding experience. Let us know if you're missing any features about SCP Cloud Foundry support from SAP Cloud SDK for Java.
+
+### Do you support SAP Cloud Platform - Neo?
```

Comment:
This is not as relevant for JavaScript, as Neo only supports Java runtimes for backend.

Reply:
Learn one thing a day they say :) I'll remove it.